assets/sass/5_layouts/_layout-general.scss

```diff
@@ -377,9 +377,11 @@
                 z-index: $toolbar - 1;
                 @include equal-radius(0);
                 background-color: $color-primary-gamma;
+                width: 100%;
 
                 @include media($medium) {
-                    @include left(64px);
```

Comment:
I believe you will need to add @include left(64px) to `layout-d` specific `listing-toolbar`